app.js

```diff
@@ -0,0 +1,30 @@
+const express = require('express');
+const { getTopics, getArticleId, getUsers, code404NotFound } = require('./controllers/news.controllers.js')
+const app = express();
+
+app.use(express.json());
```

Comment:
We arent doing any patch/post requests yet, we dont need to use express.json yet